### PR TITLE
[RC1 Ch06] VNF Test Case Requirements

### PR DESCRIPTION
**Pull request by yangyanyj**
Add initial content of VNF test case requirements, please help to review. Thanks

### CODE REVIEW

File: doc/ref_cert/lfn/chapters/chapter06.md [hunk 1]
```diff
@@ -5,19 +5,83 @@
...
+
+- Descrie and provide a Table showing Performance Profiles
+
+<a name="6.7"></a>
+## 6.7 VNF Test Case Category
+In order to Certificate the VNF, CNTT define the following four category testing which should be consistent with the VNF test category defined by OVP.
+
+|  VNF Test Case Category   |  Category Definitions  |
+| ----------------------- | ------------------|
+|  Compliance testing|  compliance check based on TOSCA using ETSI SOL004 & SOL001；OpenStack HOT using ONAP VNFREQS；GSMA profile  |
```
Comment (rabi-abdel) [inline doc/ref_cert/lfn/chapters/chapter06.md]: remove Compliance Part as this should be driven by ONAP not CNTT.

File: doc/ref_cert/lfn/chapters/chapter06.md [hunk 2]
```diff
@@ -5,19 +5,83 @@
...
 
 > we need to a list of VNFs measurements we need to collect.
 
-<a name="6.3"></a>
-## 6.3 VNF Test Cases
-> we need to have list of NFVI test cases in here.
\ No newline at end of file
+<a name="6.9"></a>
+## 6.9 VNF Test Cases
+- Compliance test cases
```
Comment (rabi-abdel) [inline doc/ref_cert/lfn/chapters/chapter06.md]: remove compliance from here too :)

**Comment by rabi-abdel** [comment]
great start @mkr1481 thank you

**Comment by mf4716** [comment]
@mkr1481 - can you please review comments, and make changes where needed, then click the Resolve Conversation (button), or request additional clarification before our RI/RC call Wed?

**Comment by rabi-abdel** [comment]
@mkr1481 please approve

**Pull request merged**